samples/bsmTx.ini

```diff
@@ -0,0 +1,133 @@
+LowerLimit = 0
+UpperLimit = 163.82
+
+[heading]
+Path = metadata.receivedMessageDetails.locationData.heading
+Type = decimal
+LowerLimit = 0
+UpperLimit = 359.9875
+
+[rxSource]
+Path = metadata.receivedMessageDetails.rxSource
+Type = enum
+Values = ["RSU", "SAT", "RV", "SNMP", "NA", "unknown"]
+EqualsValue = NA
+
+[recordGeneratedAt]
+Path = metadata.recordGeneratedAt
+Type = timestamp
+
+[recordGeneratedBy]
+Path = metadata.recordGeneratedBy
+Type = enum
+Values = ["TMC", "OBU", "RSU", "TMC_VIA_SAT", "TMC_VIA_SNMP"]
+EqualsValue = OBU
+
+[recordType]
+Path = metadata.recordType
+Type = enum
+Values = ["bsmLogDuringEvent", "rxMsg", "dnMsg", "bsmTx", "driverAlert", "unsupported"]
+EqualsValue = bsmTx
+
+[sanitized]
+Path = metadata.sanitized
+Type = enum
+Values = ["True", "False"]
+
+[schemaVersion]
+Path = metadata.schemaVersion
+Type = decimal
+EqualsValue = 6
+
+[securityResultCode]
+Path = metadata.securityResultCode
+Type = enum
+Values = ["success", "unknown", "inconsistentInputParameters", "spduParsingInvalidInput", "spduParsingUnsupportedCriticalInformationField", "spduParsingCertificateNotFound", "spduParsingGenerationTimeNotAvailable", "spduParsingGenerationLocationNotAvailable", "spduCertificateChainNotEnoughInformationToConstructChain", "spduCertificateChainChainEndedAtUntrustedRoot", "spduCertificateChainChainWasTooLongForImplementation", "spduCertificateChainCertificateRevoked", "spduCertificateChainOverdueCRL", "spduCertificateChainInconsistentExpiryTimes", "spduCertificateChainInconsistentStartTimes", "spduCertificateChainInconsistentChainPermissions", "spduCryptoVerificationFailure", "spduConsistencyFutureCertificateAtGenerationTime", "spduConsistencyExpiredCertificateAtGenerationTime", "spduConsistencyExpiryDateTooEarly", "spduConsistencyExpiryDateTooLate", "spduConsistencyGenerationLocationOutsideValidityRegion", "spduConsistencyNoGenerationLocation", "spduConsistencyUnauthorizedPSID", "spduInternalConsistencyExpiryTimeBeforeGenerationTime", "spduInternalConsistencyextDataHashDoesntMatch", "spduInternalConsistencynoExtDataHashProvided", "spduInternalConsistencynoExtDataHashPresent", "spduLocalConsistencyPSIDsDontMatch", "spduLocalConsistencyChainWasTooLongForSDEE", "spduRelevanceGenerationTimeTooFarInPast", "spduRelevanceGenerationTimeTooFarInFuture", "spduRelevanceExpiryTimeInPast", "spduRelevanceGenerationLocationTooDistant", "spduRelevanceReplayedSpdu", "spduCertificateExpired"]
+EqualsValue = success
+
+[serialId]
+Path = metadata.serialId
+Type = string
+
+[streamId]
+Path = metadata.serialId
+Type = string
+
+[bundleSize]
+Path = metadata.serialId.bundleSize
+Type = decimal
+UpperLimit = 2147483648
+LowerLimit = 1
+
+[bundleId]
+Path = metadata.serialId.bundleId
+Type = decimal
+UpperLimit = 9223372036854775807
+LowerLimit = 0
+
+[recordId]
+Path = metadata.serialId.recordId
+Type = decimal
+UpperLimit = 2147483647
+LowerLimit = 0
+
+[serialNumber]
+Path = metadata.serialId.serialNumber
+Type = serial
+#Increment = 1
```

Comment:
Should we implement or remove this `Increment = 1`?

Reply:
I'll remove it for now. Not sure why we want an increment other than 1.